teams/wg-rust-by-example.toml

```diff
@@ -1,5 +1,5 @@
 name = "wg-rust-by-example"
-subteam-of = "launching-pad"
+subteam-of = "lang-docs"
 kind = "working-group"
```

Comment:
I think while we're here making changes, it would make sense to change it to a normal team (and drop the `wg-` prefix). "working group" doesn't really make too much sense here, and everything else is a team.

(or we can do that in a separate PR)

Reply:
Makes sense.  I had almost done that originally.  Done now.